packages/block-library/src/navigation-menu-item/editor.scss

```diff
@@ -82,9 +81,9 @@
 
 	&.is-editing,
 	&.is-selected {
-		box-shadow: 0 0 1px $light-gray-900 inset;
-		border-radius: 4px;
 		min-width: 30px;
+		font-family: inherit;
+		font-size: inherit;
```

Comment:
Same comment as above here.

Reply:
yes, I removed those and some other with default inherit values.